data/mods/ssb/random-teams.ts

```diff
@@ -72,6 +72,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Hat-Trick',
 		evs: {atk: 252, def: 4, spe: 252}, nature: 'Jolly', teraType: 'Any',
 	},
+	TheJesucristoOsAma: {
+		species: 'Arceus', ability: 'The Grace Of Jesus Christ', item: ['Draco Plate', 'Dread Plate', 'Earth Plate', 'Fist Plate', 'Flame Plate', 'Icicle Plate', 'Insect Plate', 'Iron Plate', 'Meadow Plate', 'Mind Plate', 'Pixie Plate', 'Sky Plate', 'Splash Plate', 'Spooky Plate', 'Stone Plate', 'Toxic Plate', 'Zap Plate'], gender: 'N',
+		moves: ['Earthquake', 'Surf', 'Judgment'],
+		signatureMove: 'The Love Of Christ',
+		evs: {hp: 4, spa: 252, spe: 252}, nature: 'Timid',
```

Comment:
we need to hardcode arceus to not be able to terastallize in `canTerastallize`

Reply:
Fixed